simplq/src/components/pages/Admin/ShareQueue.jsx

```diff
@@ -14,7 +15,17 @@ const CopyQueue = (props) => {
   return (
     <CopyToClipboard text={shareUrl}>
       <ShareButton onClick={handleShareButtonClick} outlined={!clicked}>
-        {clicked ? 'Copied to clipboard' : 'Copy Queue Link'}
+        {clicked ? (
+          <>
+            <img src="./verified-24px.svg" alt="copied" />
+            <p>Copied to clipboard</p>
+          </>
+        ) : (
+          <>
+            <FileCopyIcon />
+            <p>Copy to clipboard</p>
+          </>
+        )}
       </ShareButton>
```

Comment:
Could you move this logic to a function in this same component and call that function here? Just so that this return statement looks cleaner.

Reply:
Other than that, it looks good.

Reply:
I've moved it to a function.